portfolio/src/main/java/com/google/sps/entities/Comment.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.entities;
+
+import lombok.*;
+
+@Builder
+@Getter
+@Setter
+public class Comment {
```

Comment:
What do you think about adding some good comments about what this object is, and maybe how it is used?

This will give better content for when someone changes it.

Reply:
That's a good idea, I'll do so.